docs/OCP_CI_Tutorials/Containers/Container_Creation_Guide.md

```diff
@@ -56,15 +63,18 @@ When creating your tests, there shouldn't be much of a change outside of keeping
 > If your tests require any variables prior to executing the tests, please work with the CSPI team ahead of time which variables you need and (if needed) how to retrieve those variables. It is possible to get a variable (i.e. the web address of the test cluster) during a previous container's execution and that variable can be placed in a file in the `SHARED_DIR` for usage in other containers.
```

Comment:
It appears that some words are missing from the following portion of this line: "please work with the CSPI team ahead of time which variables"

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)